From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 25 Mar 2014 00:31:48 +0100
Subject: systemctl: --kill-mode is long long gone,
 don't mention it in the man page

(cherry picked from commit 0088d63151e088d62104d88f866e9eb049091c22)
---
 man/systemctl.xml               | 5 ++---
 shell-completion/bash/systemctl | 5 +----
 2 files changed, 3 insertions(+), 7 deletions(-)

diff --git a/man/systemctl.xml b/man/systemctl.xml
index 1fb77ae8..e7ae3093 100644
--- a/man/systemctl.xml
+++ b/man/systemctl.xml
@@ -652,9 +652,8 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
           <listitem>
             <para>Send a signal to one or more processes of the
             unit. Use <option>--kill-who=</option> to select which
-            process to kill. Use <option>--kill-mode=</option> to select
-            the kill mode and <option>--signal=</option> to select the
-            signal to send.</para>
+            process to kill. Use <option>--signal=</option> to select
+            the signal to send.</para>
           </listitem>
         </varlistentry>
         <varlistentry>
diff --git a/shell-completion/bash/systemctl b/shell-completion/bash/systemctl
index dc7ef668..0dfc868b 100644
--- a/shell-completion/bash/systemctl
+++ b/shell-completion/bash/systemctl
@@ -74,7 +74,7 @@ _systemctl () {
                [STANDALONE]='--all -a --reverse --after --before --defaults --fail --ignore-dependencies --failed --force -f --full -l --global
                              --help -h --no-ask-password --no-block --no-legend --no-pager --no-reload --no-wall
                              --quiet -q --privileged -P --system --user --version --runtime'
-                      [ARG]='--host -H --kill-mode --kill-who --property -p --signal -s --type -t --state --root'
+                      [ARG]='--host -H --kill-who --property -p --signal -s --type -t --state --root'
         )
 
         if __contains_word "--user" ${COMP_WORDS[*]}; then
@@ -99,9 +99,6 @@ _systemctl () {
                         --kill-who)
                                 comps='all control main'
                         ;;
-                        --kill-mode)
-                                comps='control-group process'
-                        ;;
                         --root)
                                 comps=$(compgen -A directory -- "$cur" )
                                 compopt -o filenames
